lms/djangoapps/django_comment_client/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls.defaults import url, patterns, include
 
-urlpatterns = patterns('',  # nopep8
-    url(r'forum/?', include('django_comment_client.forum.urls')),
+urlpatterns = patterns(
+    '',
+    url(r'forum/', include('django_comment_client.forum.urls')),
```

Comment:
@jimabramson you probably don't have a lot of context regarding our project here, but I'm wondering if you could do a quick weigh in on this regex change and whether it might have some inadvertent consequences.